R/spat4cast_submit.R

```diff
@@ -1,8 +1,22 @@
 
-spat4cast_submit <- function(dir, theme = "lai_recovery") {
-  # upload anonymously to submissions bucket
+spat4cast_submit <- function(dir, theme = "lai_recovery", fire = "august_complex") {
+  
+  submitted_files <- list.files(dir, full = TRUE)
+  
+  # Check file type
+  assertthat::assert_that(n_distinct(tools::file_ext(submitted_files)) == 1)
+  assertthat::assert_that(unique(tools::file_ext(submitted_files)) == "tif", msg = "Error: The file extension in the directory is not 'tif'. Please check the file format.")
+  
+  # Check bounding box for files
+  ## can we use the `fire` argument to pull a template target file for comparison?
+  ## if we are to enforce these attributes, then we should modify `ingest_planetary_data` to remove dy, dx, dt, srs arguments (and have them enforced according to the fire of intered)
+  res(rast(submitted_files))
+  origin(rast(submitted_files))
+  ext(rast(submitted_files))
```

Comment:
minor but I think slightly more efficient to call `rast(submitted_files)` only once.  

Good question, we should ask folks if we want to verify some / all of these or not, and how.  (i.e. maybe we want to verify bounding box but only within a certain tolerance rather than exact equivalence.  Also we maybe we want to check the `crs` projection of the submission?  

Anyway great to have these here as placeholders, would be easy to add in future.